hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestRoundRobinVolumeChoosingPolicy.java

```diff
@@ -50,11 +52,16 @@ public class TestRoundRobinVolumeChoosingPolicy {
   private final List<HddsVolume> volumes = new ArrayList<>();
 
   private static final OzoneConfiguration CONF = new OzoneConfiguration();
-  private static final String BASE_DIR =
-      getTestDir(TestRoundRobinVolumeChoosingPolicy.class.getSimpleName())
-          .getAbsolutePath();
-  private static final String VOLUME_1 = BASE_DIR + "disk1";
-  private static final String VOLUME_2 = BASE_DIR + "disk2";
+  @TempDir
+  private static Path baseDir;
+  private static String volume1;
+  private static String volume2;
```

Comment:
Same here: please change these to non-`static` and remove manual delete in `@AfterEach`.